src/snoowrap.d.ts

```diff
@@ -74,6 +76,15 @@ declare class Snoowrap {
   getNew(subredditName?: string, options?: ListingOptions): Promise<_Listing<_Submission>>;
   getNewCaptchaIdentifier(): Promise<string>;
   getNewComments(subredditName?: string, options?: ListingOptions): Promise<_Listing<_Comment>>;
+  getNewModmailConversations(options?: ListingOptions & { entity?: string }): Promise<_Listing<_ModmailConversation>>;
+  createModmailDiscussion(options: { body: string, subject: string, srName: string }): Promise<_ModmailConversation>;
+  getNewModmailConversation(id: string): Promise<_ModmailConversation>;
+  // TODO
+  markNewModmailConversationsAsRead(convs: _ModmailConversation[]): any;
+  markNewModmailConversationsAsUnread(convs: _ModmailConversation[]): any;
+  getNewModmailSubreddits(): Promise<_Subreddit[]>;
+  getUnreadNewModmailConversationsCount(): Promise<{ highlighted: number, notifications: number, archived: number, new: number, inprogress: number, mod: number }>;
+  bulkReadNewModmail(subs: Array<_Subreddit | string>, state: 'new'|'inprogress'|'mod'|'notifications'|'archived'|'highlighted'|'all'): Promise<_Listing<_ModmailConversation>>;
```

Comment:
Feels like these should be string enums as well, probably under ModmailConversation

Reply:
We already have `conversationStates`, what should the identifier for the new enum be? AFAICT this is the only place where the string versions of the states are used.

Reply:
Might be better to keep it DRY if it's just in one place. I just like to extract as many things as possible :P

Otherwise I'd name it `filterStates` or something of the kind

Reply:
Might just be me being lazy, but I'm predisposed to keeping it as it is :P the string versions of the states are only used there, and IMO it would be confusing to have two similarly-named-but-different enums